sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/base/DependencyCheckConstants.java

```diff
@@ -23,19 +23,19 @@ public final class DependencyCheckConstants {
 
     public static final String JSON_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.jsonReportPath";
     public static final String HTML_REPORT_PATH_PROPERTY = "sonar.dependencyCheck.htmlReportPath";
-    public static final String SEVERITY_BLOCKER = "sonar.dependencyCheck.severity.blocker";
     public static final String SEVERITY_CRITICAL = "sonar.dependencyCheck.severity.critical";
```

Comment:
Critical is no longer used by SonarQube.

Reply:
Agreed, as above.